main.go

```diff
@@ -182,7 +179,7 @@ func run(ctx context.Context, name string, args, env []string, l *slog.Logger) i
 	go func() {
 		<-exitch
 		time.Sleep(killWait)
-		l.DebugContext(ctx, "Terminating unrespnsive process", "cmd", cmd.String())
+		l.WarnContext(ctx, "Terminating unrespnsive process", "cmd", cmd.String())
```

Comment:
```suggestion
		l.WarnContext(ctx, "Terminating unresponsive process", "cmd", cmd.String())
```

Reply:
🤦🏻 I'll open a new PR with this fix. Thanks for catching!